README.md

```diff
@@ -11,130 +12,129 @@ The framework will help you to:
 
 ## Getting Started
 
-The following overall structure for your charm directory is recommended:
+Charms written using the operator framework are just Python code. The intention
+is for it to feel very natural for somebody used to coding in Python, and
+reasonably easy to pick up for somebody who might be a domain expert but not
+necessarily a pythonista themselves.
 
-```
-.
-├── config.yaml
-├── metadata.yaml
-├── mod/
-├── lib/
-│   └── ops -> ../mod/operator/ops
-├── src/
-│   └── charm.py
-└── hooks/
-    ├── install -> ../src/charm.py
-    └── start -> ../src/charm.py  # for k8s charms per below
-```
-
-The `mod/` directory should contain the operator framework dependency as a git
-submodule:
+The dependencies of the operator framework are kept as minimal as possible;
+currently that's Python 3.5 or greater, and `PyYAML` (both are included by
+default in Ubuntu's cloud images from 16.04 on).
 
-```
-git submodule add https://github.com/canonical/operator mod/operator
-```
+If you're new to the world of Juju and charms, you should probably dive into our
+[tutorial](/TBD).
```

Comment:
I'm -1 to land and expose text saying "hey! please take a look at here (NO, NOT REALLY)". 

Better to hold off on those lines, add them later when we DO have a tutorial to point to.